include/cantera/thermo/LatticePhase.h

```diff
@@ -178,29 +178,6 @@ namespace Cantera
  * K_c \f$, using the second and third part of the above expression as a
  * definition for the concentration equilibrium constant.
  *
- * ## Instantiation of the Class
- *
- * The constructor for this phase is located in the default ThermoFactory for
- * %Cantera. A new LatticePhase object may be created by the following code
- * snippet:
- *
- * @code
- *    XML_Node *xc = get_XML_File("O_lattice_SiO2.xml");
- *    XML_Node * const xs = xc->findNameID("phase", "O_lattice_SiO2");
- *    ThermoPhase *tp = newPhase(*xs);
- *    LatticePhase *o_lattice = dynamic_cast <LatticPhase *>(tp);
- * @endcode
- *
- * or by the following constructor:
- *
- * @code
- *    XML_Node *xc = get_XML_File("O_lattice_SiO2.xml");
- *    XML_Node * const xs = xc->findNameID("phase", "O_lattice_SiO2");
- *    LatticePhase *o_lattice = new LatticePhase(*xs);
- * @endcode
- *
- * The XML file used in this example is listed in the next section
- *
```

Comment:
Should this be replaced by instructions for instantiation via `AnyMap`? Also, there is a subsequent XML Example that should likely be removed?

Reply:
The reason I went ahead with removing these "instantiation" examples is because I think they are poor examples of how to instantiate phases in the first place, and because nothing about this is specific to a particular `ThermoPhase` model. Good ways to do this are better documented by the C++ examples, such as https://cantera.org/examples/cxx/demo.html.

I agree that the the XML input file examples also need to be on their way out, but I didn't want to lose those just yet given that (a) the file format is still supported for this Cantera version and (b) I think we do need a home somewhere for some complete input file examples, beyond just the snippets currently available in the YAML API documentation (https://cantera.org/documentation/docs-2.5/sphinx/html/yaml/phases.html#phase-thermodynamic-models).

Reply:
I believe it’s :+1: with leaving the example as long as it is marked as ‘deprecated’